lib/gui/modules/drive-scanner.js

```diff
@@ -16,86 +16,140 @@
 
 'use strict';
 
-const Rx = require('rx');
 const _ = require('lodash');
 const EventEmitter = require('events').EventEmitter;
 const drivelist = require('drivelist');
+const debug = require('debug')('etcher:drive-scanner');
 const settings = require('../models/settings');
 
-const DRIVE_SCANNER_INTERVAL_MS = 2000;
-const DRIVE_SCANNER_FIRST_SCAN_DELAY_MS = 0;
-const emitter = new EventEmitter();
+/**
+ * @summary Time to wait in between scan calls
+ * @type {Number}
+ * @constant
+ */
+const DRIVE_SCANNER_DELAY_MS = 2000;
+
+/**
+ * @summary Time to delay the first scan (in milliseconds)
+ * @description This prevents the drive scans from kicking
```

Comment:
Interesting. I wonder if instead of hard-coding a magic number, we can listen to any Electron browser window event to trigger the loop once the window is shown, for example.